dev-haskell/editor-open/editor-open-0.6.0.0.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# ebuild generated by hackport 0.6.1.9999
+
+CABAL_FEATURES="lib profile haddock hoogle hscolour"
+inherit haskell-cabal
+
+DESCRIPTION="Open the user's $VISUAL or $EDITOR for text input"
+HOMEPAGE="https://github.com/pharpend/editor-open"
+SRC_URI="mirror://hackage/packages/archive/${PN}/${PV}/${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0/${PV}"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND=">=dev-haskell/conduit-1.2.3:=[profile?]
+	dev-haskell/conduit-extra:=[profile?]
+	dev-haskell/resourcet:=[profile?]
+	dev-haskell/temporary:=[profile?]
+	>=dev-lang/ghc-7.8.2:=
+"
+DEPEND="${RDEPEND}
+	>=dev-haskell/cabal-1.18.1.3
+"
+
+src_install() {
+	cabal_src_install
+
+	rm -r "${D}/usr/bin"
```

Comment:
Instead of deleting a binary I suggest adding `Buildable: false` to `.cabal` file to avoid it being built. r sending a patch upstream to convert it to proper cabal test suite.